lib/auth/permissions.go

```diff
@@ -273,6 +273,12 @@ func (a *authorizer) authorizeRemoteUser(ctx context.Context, u RemoteUser) (*Co
 	// Adjust expiry based on locally mapped roles.
 	ttl := time.Until(u.Identity.Expires)
 	ttl = checker.AdjustSessionTTL(ttl)
+	var previousIdentityExpires time.Time
+	if u.Identity.MFAVerified != "" {
+		prevIdentityTTL := time.Until(u.Identity.Expires)
```

Comment:
Isn't this the same as ttl?

Reply:
Yes, it was, changed to what I meant it to be: a26f201546be180648fceee39c15da2abe427c49